Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity

```diff
@@ -21074,7 +20930,7 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 1395862313}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: -0.25}
+  m_LocalPosition: {x: 0, y: 0, z: 0}
```

Comment:
I think this is either moving the camera or the playspace back to the origin. We might need to do some more work to prevent this from happening, but I think this change is reverting some of the intent of #9992

Reply:
oh woops I think some of the serialization got wonky, I'll revert this I mainly cared about fixing the colliders